File: C/lt_01_estseq/lt01_estseq_03.c
#include <stdio.h>

int main() {
  float base, altura;
  printf("\nInsira o Valor da Base: ");
  scanf("%f", &base);
  printf("\nInsira o Valor da Altura: ");
  scanf("%f", &altura);
  base *= altura;
  base /= 2;
  printf("\nO valor da Area e %.0f", base);
  return 0;
}

File: Java/LT01_ESTSEQ/LT01_ESTSEQ_03.java

/*
 * Nome do Programador: 
 * Data: 13/02/2025
 * Descrição: Receba a base e a Altura de um triângulo. Calcule e mostre a sua área.
 */
import javax.swing.JOptionPane;

public class LT01_ESTSEQ_03 {
	public static void main(String args[]) {
		Double altura;
		altura = Double.parseDouble(JOptionPane.showInputDialog("Digite o valor da altura: "));
		altura *= (Double.parseDouble(JOptionPane.showInputDialog("Digite o valor dabase: ")) / 2);
		JOptionPane.showMessageDialog(null, "A área é " + altura);
	}
}
